src/dbos-executor.ts

```diff
@@ -283,6 +309,9 @@ export class DBOSExecutor {
   }
 
   async workflow<T extends any[], R>(wf: Workflow<T, R>, params: WorkflowParams, ...args: T): Promise<WorkflowHandle<R>> {
+    if (this.debugMode) {
```

Comment:
We'll need to do the same for the transaction method I guess.

Reply:
I implemented a new `debug_workflow.ts` which includes a `transaction` method.